cloud/src/meta-service/meta_service.h

```diff
@@ -977,14 +977,23 @@ class MetaServiceProxy final : public MetaService {
             auto dist = std::uniform_int_distribution(-config::idempotent_request_replay_delay_range_ms,
                                                       config::idempotent_request_replay_delay_range_ms);
             int64_t sleep_ms = config::idempotent_request_replay_delay_base_ms + dist(rng);
+            std::string debug_string = req.ShortDebugString();
+            if constexpr (std::is_same_v<Request, GetTabletStatsRequest>) {
+                auto short_req = req;
+                if (short_req.tablet_idx_size() > 10) {
+                    short_req.mutable_tablet_idx()->DeleteSubrange(10, req.tablet_idx_size() - 10);
```

Comment:
make a copy of subrange instead of changing in place, in case of we use the request for next retry.